src/main/kotlin/com/example/noswimmingserver/domain/reading_journal/presentation/ReadingJournalController.kt

```diff
@@ -36,6 +38,15 @@ class ReadingJournalController(
         deleteReadingJournalService.execute(readingJournalId)
     }
 
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @PutMapping("/{reading-journal-id}")
+    fun editReadingJournal(
+        @PathVariable("reading-journal-id") readingJournalId: Long,
+        @RequestBody @Valid request: UpdateReadingJournalRequest,
```

Comment:
🚫 **[ktlint]** <annotation> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Multiple annotations should not be placed on the same line as the annotated construct